### ISSUE TITLE
account create error handling

### ISSUE THREAD

**Issue by mrjvs** [PretendoNetwork/website#153]
when making an account, all input validation errors are not shown to the user.
cookie with error message is being made but not being shown.

this is most likely the problem with people not being able to make accounts.

**Comment by ashmonty** [comment]
With that fixed, the error appears to be "Username contains invalid characters", which is weird considering the username I'm trying to use is `ashash`
![image](https://user-images.githubusercontent.com/49426949/191310167-ae4a9acd-bf0f-4321-8f19-5d1c0f7991d9.png)

**Comment by CorruptedPhazite** [comment]
I've noticed that it consistently triggers even if the Username or Mii Name doesn't contain invalid characters if the input validation for either the Username or Mii Name failed on a previous creation attempt. I found that I could work around this issue by simply retyping both fields

**Comment by jjok** [comment]
Yeah, I kept getting the invalid characters in username error. In the end, I filled in the form, did the captcha, then deleted and retyped the username and mii name, and that worked.

**Comment by jonbarrow** [comment]
Can anyone confirm if this is still happening for them? I believe this issue was fixed a while ago (bad regex), and I'm cleaning up some old issues
